interpreter/cling/test/Lookup/named.C

```diff
@@ -6,6 +6,7 @@
 // LICENSE.TXT for details.
 //------------------------------------------------------------------------------
 
+// clang-format off
```

Comment:
Can we put a `.clang-format` file in the `test` folder instructing clang-format to ignore that folder?